service/controller/managementcluster/resource.go

```diff
@@ -152,6 +155,19 @@ func newResources(config resourcesConfig) ([]resource.Interface, error) {
 		}
 	}
 
+	var ciliumnetpolResource resource.Interface
+	{
+		c := ciliumnetpol.Config{
+			K8sClient: config.DynamicK8sClient,
```

Comment:
Maybe call it dynamicclient or just client?

Reply:
I renamed it `DynamicK8sClient`